internal/cache/llmcache/llm_cache_test.go

```
@@ -0,0 +1,442 @@
+	if elapsed >= 250*time.Millisecond {
+		t.Fatalf("GetOrCompute should return immediately on cancellation, but elapsed %v", elapsed)
+	}
+}
+
+func TestEngine_JitterRange(t *testing.T) {
+	t.Run("100s base duration", func(t *testing.T) {
+		base := 100 * time.Second
+		minExpected := 90 * time.Second
+		maxExpected := 110 * time.Second
+
+		var minObserved time.Duration = 1000 * time.Second
```

golangci-lint (2.12.2)

[error] ST1023: should omit type time.Duration from declaration; it will be inferred from the right-hand side

(staticcheck)